dlp/api/Snippets.Tests/InspectDataWithHotwordRuleTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class InspectDataWithHotwordRuleTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public InspectDataWithHotwordRuleTests(DlpTestFixture fixture) => _fixture = fixture;
+        [Fact]
+        public void TestInspectDataWithHotwordRule()
+        {
+            string text = "Patient's MRN 444-5-22222 and just a number 333-2-33331233";
+            string customRegexPattern = "[1-9]{3}-[1-9]{1}-[1-9]{5}";
+            string hotwordRegexPattern = "(?i)(mrn|medical)(?-i)";
+            var result = InspectDataWithHotwordRule.InspectDataHotwordRule(_fixture.ProjectId, text, customRegexPattern, hotwordRegexPattern);
+            Assert.Equal(2, result.Result.Findings.Count);
```

Comment:
Please add an assert to confirm "Info type: C_MRN" is present in the output. Or equivalent assert in the "result"

Reply:
Added.